junction/proposals/management/commands/export_conf_proposals.py

```diff
@@ -0,0 +1,107 @@
+# -*- coding: utf-8 -*-
+import csv
+import os
+import sys
+
+from django.contrib.auth.models import User
+from django.core.management.base import BaseCommand
+from settings.common import ROOT_DIR
+
+from junction.base.constants import ProposalStatus
+from junction.conferences.models import Conference
+from junction.proposals.models import Proposal, ProposalSectionReviewerVoteValue
+from junction.proposals.permissions import is_conference_moderator
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--conference_slug",
+            default=None,
+            help="Enter the conference slug where to export reviewer votes from",
```

Comment:
Update the help text to `Conference slug`

Reply:
done